.gitignore

```diff
@@ -1 +1,3 @@
 .vscode/
+cli/bin
+cli/vendor
```

Comment:
why is there a vendor directory here? This should be using go mod

Reply:
I usually do `go mod vendor`. But never mind I will drop this from `gitignore`.